service/routes.py

```diff
@@ -279,22 +268,23 @@ def get(self, customer_id):
             raise NotFound(f"Customer with id '{customer_id}' was not found.")
         customer_dict = customer.serialize()
         addresses = customer_dict["addresses"]
-        if len(request.args) != 0:
-            all_query_key = ["city", "state", "country", "zipcode", "street_address"]
-            for key in request.args.keys():
```

Comment:
I'm wonderin the same thing there. If we keep the check, then test_query_customer_list_by_wrong_query() and test_wrong_query_customer_addresses() should be able to pass.

Reply:
test_wrong_query_customer_addresses() actually ends up failing because the request parser ends up ignoring the wrong query, we could access the arguments through request.args and the request parser if that's what we want